### ISSUE TITLE
Remove numbering of bibliography entries

### ISSUE THREAD

**Issue by acircleda** [mitchelloharawild/vitae#203]
Is there any way to remove the numbering of the bibliography entries?

**Comment by mitchelloharawild** [comment]
Yes, you can remove the numbering by using a different CSL file to style the bibliography.
A large collection of bibliography styles can be found here: https://github.com/citation-style-language/styles
I imagine that you'd be able to find a style which suits you there.
